assets/sass/v2/_forms.scss

```diff
@@ -26,6 +26,31 @@
         float: right;
       }
     }
+    .o-form-head.has-identifier {
+      margin-bottom: 10px;
+    }
+    .identifier-container {
+      overflow: hidden;
+      white-space: nowrap;
+      text-overflow: ellipsis;
+      text-align: center;
+    }
+    .identifier {
+      font-size: $body-font-size;
+      text-overflow: ellipsis;
+      overflow: hidden;
+      white-space: nowrap;
+      margin-bottom: 24px;
+      &.is-terminal {
+        margin-bottom: 0;
+        margin-top: -10px;
```

Comment:
instead of using a negative margin here (not ideal), maybe we can play with the positioning? i.e.
`position: relative;
  top: -10px;
`
or something similar.